swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0.0):
+        if id is None:
+            id = uuid.uuid4().int
+        self.id = id
+        self.condition = condition
+        self.category = self.get_category()
+    
+    def get_category(self):
+        return self.__class__.__name__
+    
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+    
+    def condition_description(self):
+        if self.condition == 0.0:
+            return "Mint condition"
+        elif self.condition == 1.0:
+            return "Good condition"
+        elif self.condition == 2.0:
+            return "Lightly used"
+        elif self.condition == 3.0:
+            return "Fair"
+        elif self.condition == 4.0:
+            return "Used"
+        else:
```

Comment:
Since nothing constrains `self.condition` to be an integer, how might you handle cases where a user of your class set the condition to a value like `2.5`?